src/quantum/Quantum.jsx

```diff
@@ -149,27 +149,33 @@ export default class QuantumIndex extends React.Component {
       },
       {
         title: 'Page Load tests (TP6)',
-        more: `/quantum/tp6?bits=${bits}&test=loadtime`,
-        rows: selectFrom(TP6_PAGES)
-          .where({ bits })
-          .groupBy('title')
-          .map((series, title) => (
+        more: `/quantum/tp6?bits=${bits}&test=warm-loadtime`,
+        rows: selectFrom(TP6_COMBOS)
+          .where({
+            bits,
+            test: 'warm-loadtime',
+            site: [
+              'Tp6: Facebook',
+              'Tp6: Amazon',
+              'Tp6: YouTube',
+              'Tp6: Google',
+            ],
+          })
+          .groupBy('site')
+          .map((series, site) => (
             <PerfherderGraphContainer
               // eslint-disable-next-line react/no-array-index-key
-              key={`page_${title}_${bits}`}
-              title={title}
+              key={`page_${site}_${bits}`}
+              title={site}
               series={selectFrom(series)
                 .sortBy(['ordering'])
-                .map(({ seriesConfig, ...row }) => ({
-                  ...row,
-                  seriesConfig: {
-                    and: [seriesConfig, { eq: { test: 'loadtime' } }],
-                  },
+                .map(s => ({
+                  label: s.browser,
+                  seriesConfig: s.seriesConfig,
                 }))}
             />
           ))
-          .enumerate()
-          .limit(4),
```

Comment:
In the old version, there was ?10? sites,  in popular-first order.  Since this is a summary page, the `limit(4)` was showing just the top 4 site timings. 

With all the sites added, keeping them in popular-first order was too messy: I put them in alphabetical order, and explicitly listed the 4 sites I want to show on this summary page